src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1577,6 +1577,8 @@ public static class InternalTestHooks
         internal static bool BypassOnlineHelpRetrieval;
         // It's useful to test that we don't depend on the ScriptBlock and AST objects and can use a re-parsed version.
         internal static bool IgnoreScriptBlockCache;
+        // Simulate 'System.Diagnostics.Stopwatch.IsHighResolution is false' to test Get-Uptime throw
+        internal static bool StopwatchIsHighResolutionIsFalse;
```

Comment:
How about naming it `StopwatchIsNotHighResolution`? I feel the current variable name is a bit confusing.